tests/load/locustfile.py

```diff
@@ -0,0 +1,43 @@
+import json
+import pathlib
+import random
+import threading
+
+import locust
+
+
+class LoadTest(locust.HttpUser):
+    seed_data = None
+    seeded = False
+    seed_lock = threading.Lock()
+
+    def load_seed_data(self):
+        locust_file = pathlib.Path(__file__).resolve()
+        project_root = locust_file.parents[2]  # -> goes up to the project root
+        json_path = project_root / "src" / "recordlinker" / "assets" / "test_data.json"
```

Comment:
If you use [os.path.join](https://docs.python.org/3/library/os.path.html#os.path.join) it should work for both unix and windows.